projects/observability/src/shared/components/cartesian/d3/legend/cartesian-legend.ts

```diff
@@ -12,9 +14,19 @@ import { CartesianSummaryComponent, SUMMARIES_DATA } from './cartesian-summary.c
 
 export class CartesianLegend {
   private static readonly CSS_CLASS: string = 'legend';
+  private static readonly RESET_CSS_CLASS: string = 'reset';
+  private static readonly DEFAULT_CSS_CLASS: string = 'default';
+  private static readonly ACTIVE_CSS_CLASS: string = 'active';
+  private static readonly INACTIVE_CSS_CLASS: string = 'inactive';
   public static readonly CSS_SELECTOR: string = `.${CartesianLegend.CSS_CLASS}`;
 
+  public readonly activeSeries$: Observable<Series<{}>[]>;
```

Comment:
I see that some of this was pre-existing. Since this is old code, I'm guessing this was actually from before unknown was introduced to the language, but if it's difficult to change we can defer; if it's a simple replacement, let's do it.